pre_commit/clientlib/validate_config.py

```diff
@@ -52,8 +59,19 @@ def try_regex(repo, hook, value, field_name):
 
 
 def validate_config_extra(config):
+    local_repos_found = False
     for repo in config:
-        if 'sha' not in repo:
+        if is_local_hooks(repo):
+            if local_repos_found:
+                raise InvalidConfigError(
+                    'Multiple local hooks found'
+                )
```

Comment:
one last thing (sorry, didn't notice this on the first passes), is there a reason to raise this error? it should just work now that we're doing it at the repository right? (it's a bit silly I agree, but it should work just fine if the check is removed?)